features/steps/app_steps.rb

```diff
@@ -28,6 +28,18 @@
   relaunch_crashed_app
 end
 
+When('I switch to the web browser for {int} seconds') do |duration|
+  execute_command "background", [duration.to_s]
+end
+
+When('I switch to the web browser 1 second') do |duration|
+  execute_command "background", ["1"]
+end
```

Comment:
There's a missing "for" in the sentence here, but all you need to do is put brackets around the "s" of seconds in the previous step and it makes the "s" optional.